gofer.spec

```diff
@@ -208,6 +212,7 @@ Requires: python-qpid-proton >= 0.9-5
 Provides the gofer qpid proton messaging adapter package.
 
 %files -n python-%{name}-proton
+%defattr(-,root,root,-)
```

Comment:
It is not needed for functionality, but it is needed to suppress warnings during the RPM install, such as:
```
warning: user build_user does not exist - using root
warning: group build_group does not exist - using root
```
It wouldn't be a bad idea to have defaults for files in all packages, but this was the only one I saw warnings for.